src/uct/base/uct_iface.h

```diff
@@ -7,6 +7,10 @@
 #ifndef UCT_IFACE_H_
 #define UCT_IFACE_H_
 
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
```

Comment:
why needed in this H file?
is it to not have to include it from all transports?
seems like a "band-aid" ...

Reply:
it needed in all files where macro like ```IBV_HW_TM``` is used (on line 44 below)